twitter_scraper/modules/profile.py

```diff
@@ -57,6 +58,8 @@ def __parse_profile(self, page):
 
         self.profile_photo = html.find(".ProfileAvatar-image")[0].attrs["src"]
 
+        self.banner_photo = html.find(".ProfileCanopy-headerBg img")[0].attrs["src"]
```

Comment:
It will raise KeyError if banner photo not exists

Reply:
I fixed. Is it okey?